lib/ansible/modules/network/f5/bigip_static_route.py

```diff
@@ -286,22 +295,23 @@ def destination_ip(self):
     def netmask(self):
         if self._values['netmask'] is None:
             return None
-        # Check if numeric
+        result = -1
```

Comment:
"magic value" is considered less readable and is more often used in C-level languages, which don't have features/machinery to do this as elegant as Python does. Would you consider using exceptions to communicate failures?